flixel/system/FlxBasePreloader.hx

```diff
@@ -2,30 +2,35 @@ package flixel.system;
 
 import flash.display.Bitmap;
 import flash.display.BitmapData;
+import flash.display.GradientType;
+import flash.display.GraphicsPathWinding;
+import flash.display.Shape;
+import flash.display.Sprite;
 import flash.display.StageAlign;
 import flash.display.StageScaleMode;
 import flash.events.Event;
 import flash.events.MouseEvent;
+import flash.geom.Matrix;
+import flash.geom.Rectangle;
 import flash.Lib;
 import flash.net.URLRequest;
 import flash.text.TextField;
 import flash.text.TextFormat;
 import flash.text.TextFormatAlign;
-import flixel.util.FlxColor;
 import flixel.util.FlxStringUtil;
 
 class FlxBasePreloader extends NMEPreloader
 {
 	/**
-	 * Add this string to allowedURLs array if you want to be able to test game with enabled site-locking on local machine 
+	 * Add this string to allowedURLs array if you want to be able to test game with enabled site-locking on local machine
 	 */
 	public static inline var LOCAL:String = #if flash "local" #else "localhost" #end;
-	
+
```

Comment:
Extra lines?  Are you referring to the whitespace normalization?  If so, there should not be any "extra" lines.  I will of course double-check once I get this branch rebased to address the changes you requested below as well as the conflicts due to the PR's old age.